packages/graphql-docs-generator/src/generator/utils/loading.ts

```diff
@@ -1,44 +1,54 @@
-import * as fs from 'fs';
-
-import { buildClientSchema, Source, concatAST, parse, DocumentNode, GraphQLSchema, buildASTSchema } from 'graphql';
-
-import { extname, join, normalize } from 'path';
-
-export function loadSchema(schemaPath: string): GraphQLSchema {
-  if (extname(schemaPath) === '.json') {
-    return loadIntrospectionSchema(schemaPath);
+import { buildClientSchema, Source, parse, GraphQLSchema, buildASTSchema } from 'graphql';
+import { SchemaType } from '../types';
+
+export function loadSchema(schema: string, schemaType: SchemaType): GraphQLSchema {
```

Comment:
The function name can be renamed here as it changes the signature. I think the `parseSchema` could be a better name here as `loadSchema` usually means read schema from the file path, and the `loadSchema` is already defined in amplify codegen with the same name (or the `loadSchema` in amplify codegen can be renamed to `readSchemaFromFile`). Here the schema file string is the input and the function is to parse it to an output under graphql module

Reply:
I've renamed this to `buildSchema` which I think makes more sense since, we're not only parsing the schema input but also building it and returning a AST. Thanks!